Sources/NetworkProtection/Status/ConnectionServerInfoObserver/ConnectionServerInfoObserverThroughDistributedNotifications.swift

```diff
@@ -60,6 +60,10 @@ public class ConnectionServerInfoObserverThroughDistributedNotifications: Connec
         }
     }
 
+    public func refreshServerInfo() {
+        // no-op
+    }
```

Comment:
Having to add these implementations is no good, and points to the protocol change being the wrong approach. I'm going to re-work this.